pkg/management/component.go

```diff
@@ -589,15 +589,24 @@ func (c *component) ExecuteActionsEmail(ctx context.Context, realmName string, u
 	var accessToken = ctx.Value(cs.CtContextAccessToken).(string)
 
 	var actions = []string{}
-
+	var eventRecorded bool = false
 	for _, requiredAction := range requiredActions {
 		actions = append(actions, string(requiredAction))
 		if string(requiredAction) == initPasswordAction {
 			//store the API call into the DB
 			c.reportEvent(ctx, "INIT_PASSWORD", database.CtEventRealmName, realmName, database.CtEventUserID, userID)
+			eventRecorded = true
 		}
 	}
 
+	if !eventRecorded {
```

Comment:
With this mechanism, the password init could hide some other required action.
What do you think about record the ACTION_EMAIL in any case ?

Reply:
Yes, we can do this
DONE